### PR TITLE
 npm: additional module listing use case

### PR DESCRIPTION
**Pull request by raylas**
Added a use case for listing all globally installed modules with a tree depth of 0.

### CODE REVIEW

File: pages/common/npm.md [hunk 1]
```diff
@@ -27,6 +27,10 @@
```
Comment (agnivade) [inline pages/common/npm.md]: Please change this description to `List of tree of installed modules referenced in package.json`

File: pages/common/npm.md [hunk 2]
```diff
@@ -27,6 +27,10 @@
 
 `npm list`
 
+- List globally installed modules:
+
+`npm list -g --depth=0`
```
Comment (agnivade) [inline pages/common/npm.md]: --depth={{0}}

File: pages/common/npm.md [hunk 3]
```diff
@@ -27,6 +27,10 @@
 
 `npm list`
 
+- List globally installed modules:
```
Comment (agnivade) [inline pages/common/npm.md]: `List top-level globally installed modules`

File: pages/common/npm.md (hunk 1 above)
Comment (sbrl) [reply pages/common/npm.md]: Shouldn't `package.json` be in backticks?

Comment (agnivade) [reply pages/common/npm.md]: Backticks are only for commands, not filenames like these. I also checked the yarn.md page. It also refers to package.json simply without any backticks.

Comment (sbrl) [reply pages/common/npm.md]: Ah, ok 😺

**Comment by sbrl** [comment]
Thanks, @raylas 😺

**Pull request merged**